corehq/apps/hqadmin/management/commands/shutdown_celery_worker_by_hostname.py

```diff
@@ -11,3 +13,9 @@ def handle(self, hostname, *args, **options):
         celery = Celery()
         celery.config_from_object(settings)
         celery.control.broadcast('shutdown', destination=[hostname])
+        worker_responses = celery.control.ping(timeout=10, destination=[hostname])
+        pings = parse_celery_pings(worker_responses)
+        if hostname in pings:
+            print 'Did not shutdown worker'
+            exit(1)
+        print 'Successfully initiated warm shutdown'
```

Comment:
i tested this out on staging and `shutdown` returns nothing in any situation. the ping response will return empty though if it's properly in warm shutdown